tools/arm_compiler/CROSSTOOL

```diff
@@ -3,7 +3,7 @@ minor_version: ""
 default_target_cpu: "armhf"
 
 default_toolchain {
-  cpu: "armhf"
+  cpu: "armeabi-v7a"
```

Comment:
@Hanlen It is not a bug, armhf is for arm linux, while armeabi-v7a is for android. Do you have any problems on compiling arm linux?

Reply:
I run the command "sh tools/cmake-build-arm-linux-gnueabihf-full.sh", and can not reporduct this problem. I noticed that this error was not caused by cmake but the bazel. so what is you bazel command? or the python command that invoke the bazel?

Reply:
This is my baze command which compile the arm linux:
python tools/converter.py convert --config=${CONF_FILE}  --target_socs=RK3399 --model_graph_format=file --model_data_format=file --cl_mem_type=buffer --target_abi=armhf --device_yml=${DEVICE_CONF_FILE}
The key parameter is device_yml, which represents the information of arm linux device, you can reference: https://mace.readthedocs.io/en/latest/user_guide/advanced_usage.html#run-you-model-on-the-embedded-device-arm-linux

Reply:
Thanks for your reply.
Your command works well, but I want it to output as a static lib file.
I run it like this "python tools/converter.py convert --config=${CONF_FILE} --target_abi=armhf  " 
Here is a part of  the config file :
![image](https://user-images.githubusercontent.com/7788177/59000624-e3ef3c80-883d-11e9-917c-a5bdbded8ffa.png)

Reply:
MACE can not compile a model to a static lib, if you means the static lib include only MACE engine, the command "sh tools/cmake-build-arm-linux-gnueabihf-full.sh" will meet your needs.